pkg/vcs/gitlab/mr_status_updater.go

```diff
@@ -196,6 +197,12 @@ func (p *RunStatusUpdater) getLatestPipelineID(ctx context.Context, rmd runstrea
 	return nil
 }
 
+func (p *RunStatusUpdater) mergeMRIfPossible(ctx context.Context, rmd runstream.RunMetadata) error {
+	err := p.client.MergeMR(ctx, rmd.GetMRInternalID(), rmd.GetMRProjectNameWithNamespace())
+	log.Debug().AnErr("err", err).Msg("merge MR")
+	return err
```

Comment:
if we logged this, and we don't plan on doing anything w/ the error, let's not return it